keycloak/sms-provider/src/main/java/org/sunbird/utils/SunbirdAuthUtil.java

```diff
@@ -0,0 +1,49 @@
+package org.sunbird.utils;
+
+import java.util.ArrayList;
+import java.util.List;
+import org.jboss.logging.Logger;
+import org.keycloak.authentication.AuthenticationFlowContext;
+import org.keycloak.models.KeycloakSession;
+import org.keycloak.models.UserModel;
+import org.keycloak.models.utils.KeycloakModelUtils;
+import org.sunbird.keycloak.resetcredential.sms.KeycloakSmsAuthenticatorConstants;
+
+/**
+ * 
+ * @author Amit Kumar
+ * 
+ *         Utility class for keycloak, it contains all the common method used across the
+ *         application.
+ * 
+ */
+public class SunbirdAuthUtil {
```

Comment:
Duplicate class...

Reply:
@kochhar  I made the changes suggested by you, please review and let me know if any more changes required